contracts/WildcardSteward_v3.sol

```diff
@@ -551,7 +565,7 @@ contract WildcardSteward_v3 is Initializable {
 
             if (
                 patronageOwedByTokenPatron > 0 &&
-                patronageOwedByTokenPatron >= deposit[tokenPatron]
+                patronageOwedByTokenPatron > deposit[tokenPatron]
```

Comment:
I mentioned this on slack :)

Reply:
Yeah think we are both onboard with this one.